lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1091,6 +1091,18 @@ def determine_group_level_monitored_resource_and_labels(tag,
         common_labels.delete("#{COMPUTE_CONSTANTS[:service]}/resource_name")
       end
 
+      # Cloud Dataflow and Cloud ML.
+      # These labels can be set via configuring 'labels'.
```

Comment:
s/`configuring 'labels'`/`the 'labels' option`/.

Reply:
Changed.